src/components/dashboard/Dashboard.js

```diff
@@ -676,6 +676,7 @@ const Dashboard = props => {
   // so we need to imitate onMomentumScrollEnd for web
   const onScroll = Platform.select({
     web: handleScrollEndDebounced,
+    ios: handleScrollEnd,
```

Comment:
1. platform `ios` is **native** ios app. You have to use isIOSWeb instead (see platform.js)
2. use debounce hook from here [https://github.com/xnimorz/use-debounce#debounced-callbacks](https://github.com/xnimorz/use-debounce#debounced-callbacks)